sensor/common/networkflow/manager/manager_impl_test.go

```diff
@@ -879,7 +879,7 @@ func (b *sendNetflowsSuite) TestUpdatesGetBufferedWhenUnread() {
 
 	// four times without reading
 	for i := 0; i < 4; i++ {
-		ts := timestamp.NowMinus(time.Duration(4-i) * time.Hour)
+		ts := protoconv.NowMinus(time.Duration(4-i) * time.Hour)
```

Comment:
We could reverse the for loop to start at 4 and count down to 1 instead using `4-i`.